templates/menu.tpl

```diff
@@ -30,42 +28,67 @@
                     {if $authentication_has_role.global_admin}
                         {strip}
                             <li class="dropdown">
-                                <a class="btn navbar-btn dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
-                                   aria-expanded="false" href="{#url_list_admin#}"><span class="glyphicon glyphicon-list" aria-hidden="true"></span> {$PALANG.pAdminMenu_list_admin} <span
+                                <a class="btn navbar-btn dropdown-toggle" data-toggle="dropdown" role="button"
+                                   aria-haspopup="true"
+                                   aria-expanded="false" href="{#url_list_admin#}"><span
+                                            class="glyphicon glyphicon-list"
+                                            aria-hidden="true"></span> {$PALANG.pAdminMenu_list_admin} <span
```

Comment:
All these whitespace changes make this PR hard to read. Maybe you can split them out to a separate PR?